src/components/users/LoginForm.js

```diff
@@ -30,65 +41,54 @@ const LoginForm = () => {
       dispatch(userActions.login(email, password));
     }
   };
-  const loggingIn = useSelector((state) => state.authentication.loggingIn);
-  const showLoginAlert = isSubmitted && !loggingIn && !cleanAlert && alert;
+
   return (
-    <form align="center" className="login-form" onSubmit={handleSubmit}>
-      <p className="login-form__text">EMAIL</p>
+    <form align="center" className="user-form" onSubmit={handleSubmit}>
+      <p className="user-form__text">EMAIL</p>
       <input
-        className="login-form__input"
+        className="user-form__input"
         type="email"
         name="email"
         value={email}
         onChange={handleChange}
       />
       {isSubmitted && !email && (
-        <div className="login-form__alert">
+        <div className="user-form__alert">
           <FormattedMessage id="loginform.missing.email.text" />
         </div>
       )}
-      <p className="login-form__text">PASSWORD</p>
+      <p className="user-form__text">PASSWORD</p>
       <input
-        className="login-form__input"
+        className="user-form__input"
         type="password"
         name="password"
         value={password}
         onChange={handleChange}
         autoComplete="on"
       />
       {isSubmitted && !password && (
-        <div className="login-form__alert">
+        <div className="user-form__alert">
           <FormattedMessage id="loginform.missing.pass.text" />
         </div>
       )}
       <div>
-        <button type="submit" className="login-form__btn-text">
+        <button type="submit" className="user-form__btn-text">
           <FormattedMessage id="loginform.signin.text" />
         </button>
       </div>
       {loggingIn && (
         <Loader type="ThreeDots" color="#2FBCF7" height={80} width={50} />
       )}
       {showLoginAlert && (
-        <div className="login-form__alert"> {alert.message} </div>
+        <div className="user-form__alert"> {alert.message} </div>
       )}
       {/* href="/" until de feature is done */}
-      <div className="login-form__forgot-pwd">
+      <div className="user-form__forgot-pwd">
         <a href="/">
           <FormattedMessage id="loginform.forgotpwd.text" />
         </a>
       </div>
-      <div className="login-form__facebook">
-        <a href="/">
-          <FormattedMessage id="loginform.connectfb.text" />
-        </a>
-        <hr className="login-form__hr" />
-      </div>
-      <div className="login-form__sign-up">
-        <a href="/">
-          <FormattedMessage id="loginform.signup.text" />
-        </a>
-      </div>
+      <SignUpOptions />
```

Comment:
I think much of this stuff should be outside of the form tag.

Reply:
Can you explain a little bit more?